capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module

```diff
@@ -68,3 +68,39 @@ function c4m_content_group_preprocess_block(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ * Hide fields for authenticated users and set submit function.
+ *
+ * @param $form
+ * @param $form_state
+ * @param $form_id
+ */
+function c4m_content_group_form_group_node_form_alter(&$form, &$form_state, $form_id) {
+  global $user;
+  if (!in_array('anonymous user', $user->roles)
```

Comment:
Use ``if (!$account->uid)`` to check if user is anonymous.
Use ``user_access('administer site', $account)`` to check if user is administrator.